staging/src/k8s.io/cloud-provider/app/controllermanager.go

```diff
@@ -66,7 +66,7 @@ const (
 )
 
 // NewCloudControllerManagerCommand creates a *cobra.Command object with default parameters
-// initFuncConstructor is a map of named controller groups (you can start more than one in an init func) paired to their InitFuncConstructor.
+// initFuncConstructor is a map of named controller groups paired to their InitFuncConstructor.
```

Comment:
"map of name controller groups to ... InitFuncConstructor" makes me thinks the key is either a group of controllers or a name for a group of controllers. Might be better to indicate a key is the external controller name (as defined by controllers flag in https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/#options or the CCM equivalent)